### PR TITLE
ScrollTo tests

### PR DESCRIPTION
**Pull request by stewartyu**
### Issues
#104 - ScrollTo.js tests

This is a basic functional test to ensure that it scrolls down on the page. 

There are a few issues with the `ScrollTo` plugin, but I'll open up separate issues for them. Tests can be expanded when more of these issues are sorted out.
### Steps to Reproduce

Run `npm test`.

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -89,7 +89,15 @@ module.exports = function(config) {
...
     // start these browsers
     // available browser launchers: https://npmjs.org/browse/keyword/karma-launcher
-    browsers: ['PhantomJS'],
+    browsers: ['Chrome_Sized'],
+
+
+    customLaunchers: {
+      'Chrome_Sized': {
+        base: 'Chrome',
+        flags: ['--window-size=400,600']
```
Comment (misscs) [inline karma.conf.js:98]: Does phantom have the ability to size the window?

File: src/animation/ScrollTo.js [hunk 2]
```diff
@@ -11,6 +11,8 @@ export const ScrollTo = {
    * @param {number} [offset=0] - Scroll target offset
    * @param {number} [speed=1000] Speed of animation in ms
    * @returns {void}
+   *
+   * @todo Setting speed: 0 defaults back to 1000. Setting any speed value doesn't actually change visible speed.
```
Comment (misscs) [inline src/animation/ScrollTo.js:15]: @josephco Any thoughts on why the speed is not responding to passed in arguments?

File: karma.conf.js (hunk 1 above)
Comment (stewartyu) [inline karma.conf.js:98]: @misscs It does - I just couldn't get it to work. Let me try again though because I do agree that would be the best way to go.

Comment (misscs) [inline karma.conf.js:98]: Sounds good. If you can't get it to work, just open up an issue to keep
track of what remains
